dev/com.ibm.websphere.appserver.features/visibility/public/concurrent-3.1/io.openliberty.concurrent-3.1.feature

```diff
@@ -43,7 +43,7 @@ Subsystem-Name: Jakarta Concurrency 3.1
   io.openliberty.concurrent.internal,\
   io.openliberty.concurrent.internal.compat31,\
   io.openliberty.threading.internal.java21; require-java:="21"
-kind=noship
+kind=beta
 edition=full
```

Comment:
Should `edition` be set to `core` here as well?

Also, do you need to promote all of the private feature dependencies to beta as well at the same time? (I would have expected so, but it's a while since I beta'd a feature)

Reply:
Based on the build failure in the feature checking unit tests, it looks like the answer is yes, things are more stringent these days and these things are now being enforced.  I updated the commit with the other private features and autofeature and will resubmit the build. Hopefully I found everything.

Reply:
Build and feature tests are successful after making this update. Thanks for pointing it out!